reference/constraints/When.rst

```diff
@@ -163,14 +163,16 @@ validation of constraints won't be triggered.
 To learn more about the expression language syntax, see
 :doc:`/reference/formats/expression_language`.
 
-Depending on how you use the constraint, you have access to 1 or 2 variables
+Depending on how you use the constraint, you have access to 1 or 3 variables
 in your expression:
```

Comment:
```suggestion
Depending on how you use the constraint, you have access to different variables
in your expression:
```
To match with text in doc for Expression constraint, "to 1 or 3 variables" doesn't feel clear to me

Reply:
What about « multiple internal variables »